.github/workflows/e2e-cypress-authentication.yml

```diff
@@ -81,6 +81,7 @@ jobs:
           CYPRESS_auth_widget_demo_delete_user_scope: ${{ vars.DELETE_USER_ENDPOINT_AUTHENTICATION_SCOPES }}
           CYPRESS_auth_client_id: ${{ vars.AUTHENTICATION_CLIENT_ID }}
           CYPRESS_auth_domain: ${{ vars.AUTHENTICATION_DOMAIN }}
+          CYPRESS_delete_mx_user_after_test: true
```

Comment:
I had to make this change because of another change I made to the errorHandler so that it would return more accurate error status codes. When I made that change and cypress was trying to delete MX users after each test when a lot of the tests don't have MX credentials, it was getting a 401 error which was unexpected and causing a lot of the tests to fail. I could have made it so the tests expected the 401 error but I thought it'd be better to tell the code not to attempt delete if it's not necessary.